src/content/self-hosted/eks.mdx

```diff
@@ -31,9 +31,10 @@ We recommend that you follow Amazon's [cluster creation guide](https://docs.aws.
 Okteto supports Kubernetes versions 1.20 through 1.22.
 
 We recommend the following specs:
+
 - v1.22
-- A pool with at least 3 `m4.xlarge` nodes
-- 100 GB per disk
+- A pool with at least 3 nodes (4CPUs and 32GBs each)
```

Comment:
I think this is a lot, and might scare people away. Do you think we can give guidance based on the expected size of the team?

Reply:
or maybe give both minimum and recommended ?

Reply:
I will undo those changes to reduce moving parts in this release. After the release, I would like to keep iterating on installation guides